allenact_plugins/manipulathor_plugin/manipulathor_environment.py

```diff
@@ -258,27 +324,88 @@ def close_enough(self, current_obj_pose, init_obj_pose, threshold):
             for k in ["x", "y", "z"]
         ]
         position_is_close = sum(position_close) == 3
-        rotation_close = [
-            abs(current_obj_pose["rotation"][k] - init_obj_pose["rotation"][k])
-            <= threshold
-            for k in ["x", "y", "z"]
-        ]
-        rotation_is_close = sum(rotation_close) == 3
-        return position_is_close and rotation_is_close
-
-    def get_objects_moved(self, initial_object_locations):
-        current_object_locations = self.get_current_object_locations()
+        return position_is_close
```

Comment:
@ehsanik These changes amount to backwards compatible changes to the ArmPointNav task, i.e. using this new version of the ManipulaTHOR plugin will mean we can no longer reproduce the exact results from the paper. Any thought on how we should handle this?

Reply:
I think given that the new THOR commits result in the object jittering and the success without disturbance is no longer reproducible with the new commits, it's unavoidable to break backward compatibility.

Reply:
@ehsanik - My main worry is that people will go to the ManipulaTHOR repository and end up using the old code. What do you think about me:
1. Moving the current manipulathor code to a branch called something like `cvpr-2021`.
2. Creating a new PR to manipulathor/main so that it uses these new changes. 
3. Adding some documentation to manipulathor/main readme that points people wanting to reproduce the CVPR results to the `cvpr-2021 branch`?

Reply:
Oh, I see your point. Yes, I think we can definitely do that. I can take care of it too.